From 639ebe0f4febbd068f5a4e838783f687891ddba1 Mon Sep 17 00:00:00 2001
From: De Huo <De.Huo@windriver.com>
Date: Fri, 2 Aug 2019 11:17:11 +0800
Subject: [PATCH] Fix rule accounts_passwords_pam_tally2_interval remediation
 failure

Update accounts_passwords_pam_tally2_interval rule remediation script to fix
remediate failure problem.

Upstream-Status: Pending

Signed-off-by: De Huo <De.Huo@windriver.com>
---
 .../accounts_passwords_pam_tally2_interval/bash/wrlinux1019.sh          | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_interval/bash/wrlinux1019.sh b/linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_interval/bash/wrlinux1019.sh
index e0dc6a0..ae5e445 100644
--- a/linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_interval/bash/wrlinux1019.sh
+++ b/linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_interval/bash/wrlinux1019.sh
@@ -4,7 +4,7 @@
 populate var_accounts_passwords_pam_faillock_fail_interval
 
 if grep -q '^[^#].*pam_tally2.so' /etc/pam.d/common-auth; then
-  if grep -q 'lock_time=' /etc/pam.d/common-auth; then
+  if grep -q ' lock_time=' /etc/pam.d/common-auth; then
 	  sed -i "s/lock_time=[0-9]*/lock_time=$var_accounts_passwords_pam_faillock_fail_interval/" /etc/pam.d/common-auth
   else
 	  sed -i "/auth.*pam_tally2.so/s/$/ lock_time=$var_accounts_passwords_pam_faillock_fail_interval/" /etc/pam.d/common-auth
-- 
1.9.1

